src/Uno.UI.RemoteControl/HotReload/MetadataUpdater/HotReloadAgent.cs

```diff
@@ -13,6 +13,8 @@
 using System.Reflection;
 using Uno;
 using Uno.UI.Helpers;
+using Newtonsoft.Json.Linq;
+using Uno.UI.RemoteControl.Messages;
```

Comment:
Just some typos were fixed in these files. Seems these new `usings` are not required. I guess.